src/pages/product-list/components/display/components/display-header/components/desktop-filter/desktop-filter.tsx

```diff
@@ -2,22 +2,38 @@ import React, { useState } from 'react';
 
 import { Grid, ButtonGroup, FormControl, Hidden, Typography } from '@material-ui/core';
 import { StyledButton, StyledSelect, StyledButtonGroup } from './desktop-filter.styles';
-import { OrderTypes } from 'lib/enums';
+import { OrderTypes, ListTypes } from 'lib/enums';
 import { getTextForOrderType } from './desktop-filter.utils';
 import { translate } from 'lib/translate';
+import { AppDispatch } from 'index';
+import ApplicationState from 'store/application-state';
+import { connect } from 'react-redux';
+import { changeListType } from 'store/list-type-store';
+import { listenerCount } from 'process';
 
-const DesktopFilter = () => {
+interface IProps {
+  listType: ListTypes;
+  onListTypeChange: (type: ListTypes) => void;
+}
+
+const DesktopFilter = (props: IProps) => {
   const [orderType, setOrderType] = React.useState(OrderTypes.PriceAscending);
   const [isToggled, setToggled] = useState(false);
```

Comment:
remove this isToggled state, we don't need it

Reply:
Actually we do need it because the CSS wasn't picking up the values of the "isToggled={props.listType === ListTypes.Products}"